datasets/mimiciii_demo/pipelines/mimiciii_demo/pipeline.yaml

```diff
@@ -0,0 +1,309 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# requestations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: admissions
+    description: ""
```

Comment:
Can we add column descriptions for the columns, even if they are obvious, and just a few words?

Reply:
My bad, sure.